test/testdata/namer/has_attached_class_subclass_class.rb

```diff
@@ -0,0 +1,6 @@
+# typed: true
+
+class A < Class
+  extend T::Generic
+  has_attached_class! # error: can only be used inside a `module`
```

Comment:
This error is bad but it's fine because `A < Class` is illegal anyways, so no one is going to try to do this and expect a good error message.